ci/verify-artifacts.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import configparser
+import os
+import subprocess
+import sys
+import tempfile
+from distutils.version import StrictVersion
+
+import requests
+from github import Github
+
+platform = sys.platform
+ext = ''
+
+if platform == 'win32':
+    platform = 'windows'
+    ext = '.exe'
+
+g = Github(os.environ.get("GITHUB_TOKEN"))
+
+dcos_cli_repo = g.get_repo("dcos/dcos-cli")
+
+latest_0_5 = "0.5.0"
```

Comment:
Could we use PyGithub to get the three latest big releases and then loop over it so that we don't have to update the script once we release a CLI 0.8?

Reply:
While looking at it, I think it'd make it too complex and not readable to dynamically get the latest 3 and map them to DC/OS versions, especially given that 0.8 should be the latest version pinned to a specific DC/OS version, I've just added a block to uncomment.